python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -455,6 +455,12 @@ def transform(self, df):
 
                 elif (self.strategy == 'entity_embedding'):
 
+                    def get_embeddings(x, col):
```

Comment:
Idk.
Looks good but you forgot to add the `i` variable as a parameter of your function.

I think you could even move it higher in the hierarchy, maybe as a private function throwing an error if the strategy is not "entity_embedding"
@AxeldeRomblay What do you think?